zabbix-probes/zabbix-probes-common/src/main/java/com/indigo/zabbix/utils/KeystoneClient.java

```diff
@@ -18,7 +18,12 @@
   private final KeystoneTokenProvider keystoneClient;
 
   public KeystoneClient(String location) {
-    this.keystoneClient = ProbeClientFactory.getClient(KeystoneTokenProvider.class, location);
+	  if (location.endsWith("/v3")) {
+		location = location.replace("/v3", "");
```

Comment:
Because the url that gets there (retrieved previously from CMDB) has v3 double pasted like: /../v3/v3/...